__tests__/app.test.js

```diff
@@ -309,4 +309,20 @@ describe.only("DELETE /api/comments/:comment_id", ()=>{
       expect(text).toBe('Please provide integer for comment_id')
     })
   })
+})
+describe.only("GET /api/users", ()=>{
+  test("200: returns array of users with required properties", ()=>{
+    return request(app)
+    .get('/api/users')
+    .expect(200)
+    .then(({body:{users}})=>{
+      users.forEach((user)=>{
```

Comment:
You need to check the length of the array before doing a forEach, if the array is empty this forEach will be skipped along with the expect statements making this a false positive.